src/components/DealRepaymentSchedule.tsx

```diff
@@ -19,13 +22,21 @@ export const DealRepaymentSchedule: FunctionComponent<DealRepaymentScheduleProps
 	deal,
 }) => {
 	const intl = useIntl();
+	const locales = useLocales();
 	const [showDetails, setShowDetails] = useState(false);
+	const goLiveAt = deal.goLiveAt * 1000;
 
 	className = classNames([className, "space-y-6"]);
 
 	const repaymentScheduleComponent = (repaymentSchedule: RepaymentSchedule) => {
 		const { graphData, dataSource } = generateGraphAndTableData(
-			repaymentSchedule.periods.map((p) => ({
+			repaymentSchedule.periods.map((p, index) => ({
+				day: goLiveAt
+					? formatDate(
+							add(goLiveAt, { days: (index + 1) * DAYS_IN_REPAYMENT_PERIOD }),
```

Comment:
we already have dayjs as a dependency in the program repository. I took a look at both and I don't have a strong preference but I do prefer just having one date library to work with. you can choose which one.

Reply:
We'll go for dayjs as it's more lightweight. Ill update the PR